lib/usdUfe/ufe/StagesSubject.cpp

```diff
@@ -338,6 +339,18 @@ void processAttributeChanges(
                         metadataKeys.insert(newMetadataKey);
                     }
                 }
+            } else if (infoChanged.first == SdfFieldKeys->AllowedTokens) {
```

Comment:
USD has SDF/SDR metadata for 4 of the regular UI metadata. The USD MaterialX reader does copy those [metadata as SDF attributes](https://github.com/PixarAnimationStudios/OpenUSD/blob/release/pxr/usd/usdMtlx/reader.cpp#L550) instead of Sdr metadata, so we need to add code in multiple locations to handle those SDF metadata and in the case here, send the right UFE notification.